### ISSUE TITLE
Clarify nested components

### ISSUE THREAD

**Issue by oyeanuj** [rt2zz/react-sticker#2]
Hi @rt2zz, thanks for this plugin! I have been looking for something like this myself. Quick clarification on the following -

> Multiple independent components will work as siblings but not if nested.

Do you mean multiple Sticker components? And does it matter how nested the Sticky components are within the single Sticker?

**Comment by rt2zz** [comment]
Yes, you can use multiple sticker components as long as they are not nested inside of each other. Regarding sticky components, I am actually not sure what would happen if you nest those. 

Typically all `Sticky` components would be siblings, and direct children of the `Sticker`, but I think it should work fine even if the Stickys are nested.
